aptos-move/block-executor/src/task.rs

```diff
@@ -58,12 +58,29 @@ pub trait ExecutorTask: Sync {
     fn init(args: Self::Argument) -> Self;
 
     /// Execute a single transaction given the view of the current state.
-    fn execute_transaction(
+    /// This function is only run during parallel execution.
+    /// BlockSTM will first call this function with aggregators enabled.
```

Comment:
This part of the comment (from here below) is outdated, let's just add a TODO: parallel execution has to ensure to follow the same semantics w. aggregators (e.g. enable/disable) as it happens during the sequential execution.